test/e2e/user_workloads_test.go

```diff
@@ -38,73 +37,65 @@ func (s *userWorkloadsTestSuite) TestIdlerAndPriorityClass() {
 	for _, p := range podsToIdle {
-		err := memberAwait.WaitUntilPodsDeleted(p.Namespace, wait.WithPodName(p.Name))
-		require.NoError(s.T(), err)
+		memberAwait.WaitUntilPodsDeleted(s.T(), p.Namespace, wait.WithPodName(p.Name))
 	}
 	// check notification was created
-	_, err = hostAwait.WaitForNotificationWithName("test-idler-dev-idled", toolchainv1alpha1.NotificationTypeIdled, wait.UntilNotificationHasConditions(Sent()))
-	require.NoError(s.T(), err)
+	hostAwait.WaitForNotificationWithName(s.T(), "test-idler-dev-idled", toolchainv1alpha1.NotificationTypeIdled, wait.UntilNotificationHasConditions(Sent()))
 
 	// make sure that "noise" pods are still there, and notification is not created for stage namespace
-	_, err = memberAwait.WaitForPods(idlerNoise.Name, len(podsNoise), wait.PodRunning(), wait.WithPodLabel("idler", "idler"), wait.WithSandboxPriorityClass())
-	require.NoError(s.T(), err)
-	_, err = memberAwait.WaitForPods("workloads-noise", len(externalNsPodsNoise), wait.PodRunning(), wait.WithPodLabel("idler", "idler"), wait.WithOriginalPriorityClass())
+	memberAwait.WaitForPods(s.T(), idlerNoise.Name, len(podsNoise), wait.PodRunning(), wait.WithPodLabel("idler", "idler"), wait.WithSandboxPriorityClass())
+	memberAwait.WaitForPods(s.T(), "workloads-noise", len(externalNsPodsNoise), wait.PodRunning(), wait.WithPodLabel("idler", "idler"), wait.WithOriginalPriorityClass())
 	require.NoError(s.T(), err)
-	_, err = hostAwait.WithRetryOptions(wait.TimeoutOption(10*time.Second)).WaitForNotificationWithName("test-idler-stage-idled", toolchainv1alpha1.NotificationTypeIdled, wait.UntilNotificationHasConditions(Sent()))
-	require.True(s.T(), errors.IsNotFound(err))
+	hostAwait.WithRetryOptions(wait.TimeoutOption(10*time.Second)).WaitForNotificationWithName(s.T(), "test-idler-stage-idled", toolchainv1alpha1.NotificationTypeIdled, wait.UntilNotificationHasConditions(Sent()))
 
 	// Check if notification has been deleted before creating another pod
-	err = hostAwait.WaitUntilNotificationWithNameDeleted("test-idler-dev-idled")
-	require.NoError(s.T(), err)
+	hostAwait.WaitUntilNotificationWithNameDeleted(s.T(), "test-idler-dev-idled")
 
 	// Create another pod and make sure it's deleted.
 	// In the tests above the Idler reconcile was triggered after we changed the Idler resource (to set a short timeout).
 	// Now we want to verify that the idler reconcile is triggered without modifying the Idler resource.
 	// Notification shouldn't be created again.
-	pod := s.createStandalonePod(idler.Name, "idler-test-pod-2")    // create just one standalone pod. No need to create all possible pod controllers which may own pods.
-	_, err = memberAwait.WaitForPod(idler.Name, "idler-test-pod-2") // pod was created
+	pod := s.createStandalonePod(idler.Name, "idler-test-pod-2")  // create just one standalone pod. No need to create all possible pod controllers which may own pods.
+	memberAwait.WaitForPod(s.T(), idler.Name, "idler-test-pod-2") // pod was created
```

Comment:
I guess the require.NoError on the next line was supposed to be removed?

Reply:
yes, indeed